src/messaging/ExchangeMgr.h

```diff
@@ -175,6 +175,9 @@ class DLL_EXPORT ExchangeManager : public SecureSessionMgrDelegate
     State mState;
     SecureSessionMgr * mSessionMgr;
     ReliableMessageManager mReliableMessageMgr;
+    // TODO: allow temporary propagate SecureSessionMgrDelegate events to device controller, it won't be necessary after
```

Comment:
Can we have an issue linked here please?

Reply:
Added the link to issue #4170